tests/unit/main_html5/underflow-tests.js

```diff
@@ -457,6 +458,20 @@ describe('main_html5 chrome underflow tests', function () {
     expect(_.contains(vtc.notified, vtc.interface.EVENTS.WAITING)).to.be(true);
   });
 
+  it('should raise buffered event after waiting on MS Edge when currentTime has progressed', function(){
+    OO.isChrome = false;
```

Comment:
@mityaha I think that this is already handled by the `beforeEach` above. I think that `isChrome` is set to true by default because it is used in most tests, so that's why I need to set it to false here. This test doesn't depend on the previous one to set its values though.